NAMESPACE

```diff
@@ -25,6 +26,7 @@ export(none)
 export(process_gs)
 export(process_policy)
 export(raw_over_max)
+export(read_files)
```

Comment:
It looks like we're still exporting `read_gs()`. Is that still in use or can it be removed?

Reply:
Nevermind - I just found your note further down about backwards compatability, which is a good point.